lib/puppet/ssl/host.rb

```diff
@@ -547,7 +552,9 @@ def load_certificate_bundle(bundle_string)
   # @return nil
   def download_and_save_crl_bundle(store=nil)
     begin
-      client = store ? http_client(ssl_store: store) : http_client
+      client = store ?
```

Comment:
I think multi-line ternaries are discouraged.

Reply:
most of it's the same between choices so the tenary should be shrunk to only select the argument anyway